make/kube

```diff
@@ -24,3 +27,4 @@ fissile build "${BUILD_TARGET}" \
     --output-dir="${BUILD_TARGET}" \
     --defaults-file="$(echo "${ENV_FILES}" | tr ' ' ,)" \
     --use-memory-limits=false
+git checkout env/defaults.env
```

Comment:
Jan's comment applies here too, i.e. temp file over undoing whatever local changes the user had.